bin/openosp-setup.sh

```diff
@@ -29,6 +29,7 @@ fi
 
 echo "Cloning in order to bootstrap db."
 docker-compose -f docker-compose.admin.yml run builder ./bin/clone.sh
+./bin/expedius-install.sh
```

Comment:
We want expedius optional, right? Should I add a flag or ENV var input to enable/disable it?

Reply:
Every clinic needs Expedius. It's a basic offering.